img/land-see/structure/accent-green-up.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Thanks for the svgo suggestion, @camelburrito. I went ahead and committed the optimized svgs.